modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -290,6 +296,26 @@ Button Hover Text Color (Optional):: Specifies the color of button hover text. D
 Headline Font Family (Optional):: Specifies the headline font family.
 Error Color (Optional):: Specifies the color of error messages.
 
+== API Custom Field Table
+
+The *API Custom Field Table* component shows all of an API's custom fields and their values.
+
+You can specify values for the following parameters of the *API Custom Field Table*:
+
+Record Id:: Specifies the API. Default is `{!recordId}`, which shows the instances of the API selected on the current page.
+Show Table Header:: Shows the table header.
+
+By default, the table shows all custom fields listed in the Anypoint organization. To show only some fields, open *Setup* > *Custom Metadata Type* > *ACM General Configuration Metadata* > *Manage Records*, next to *ACM Configuration* click *Edit*, and in the *CustomFieldsFilter* box enter a custom fields key such as `email,rating,case`.
```

Comment:
you can get all the custom fields key from your org using this query in the develop console. 
SELECT acm_pkg__TagKey__c FROM acm_pkg__AnypointCustomFields__x

Reply:
@wwarner-salesforce

Reply:
Thank you very much! I added this info.